src/core/internal/dassert.d

```diff
@@ -156,10 +156,10 @@ private string miniFormat(V)(const ref V v)
         string msg = "[";
         foreach (k, ref val; v)
         {
-            if (i++ > 0)
```

Comment:
Unrelated change?

Reply:
Yeah, noticed this inconsistency (would have only shown first 29 items) and fixed it.